Makefile

```diff
@@ -18,7 +18,7 @@ Makefile.common: Makefile.common.$(MAKE_BRANCH)
 Makefile.common.$(MAKE_BRANCH):
 	# Clean up any files downloaded from other branches so they don't accumulate.
 	rm -f Makefile.common.*
-	curl --fail $(MAKE_REPO)/Makefile.common -o "$@"
+	curl -k --fail $(MAKE_REPO)/Makefile.common -o "$@"
```

Comment:
What is this change for?

Reply:
My internet needs the proxy to go out through, and  `-k` helps me get rip of errors pop out every time I run make.  That was my bad not removing it in the patch.